storage-proofs/src/drgraph.rs

```diff
@@ -60,6 +71,83 @@ pub trait Graph<H: Hasher>: ::std::fmt::Debug + Clone + PartialEq + Eq {
         }
     }
 
+    /// Builds a merkle tree based on the given data and stores it in `path`
+    /// (if set).
+    // FIXME: If the `path` is set to `None` should we still create a temporary
+    //  file somewhere? (at the moment we don't)
```

Comment:
I think the answer to this question is 'yes', but I might be missing an implication.

Reply:
Yes, those `FIXME`s have been deprecated (we already implement that), I'll remove them.